### PR TITLE
FEATURE: Expose neos-ui-views

### PR DESCRIPTION
**Pull request by nlx-lars**
**What I did**
I exposed neos-ui-views so the dataLoader can be used in custom DataSource Views

**How I did it**
I added a shim in neos-ui-extensibility, a webpack alias and added that to the apiExposureMap.js.

### CODE REVIEW

File: packages/neos-ui/src/index.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ createConsumerApi(manifests, apiExposureMap);
 require('./manifest');
 require('@neos-project/neos-ui-contentrepository');
 require('@neos-project/neos-ui-editors');
-require('@neos-project/neos-ui-views');
+require('@neos-project/neos-ui-views/src/manifest');
```
Comment (dimaip) [inline packages/neos-ui/src/index.js]: Why is this necessary? Couldn't it be just `require('@neos-project/neos-ui-views');`? Then you'd need to export needed views from `index.js`.

Comment (nlx-lars) [reply packages/neos-ui/src/index.js]: That's how it's done for @neos-project/neos-ui-i18n a few lines below that as well. I took that as a reference.

Comment (dimaip) [reply packages/neos-ui/src/index.js]: Ok, never mind then, could always be refactored.
